src/script_opt/CPP/RuntimeOps.h

```diff
@@ -41,6 +49,20 @@ inline ValPtr invoke__CPP(Func* f, std::vector<ValPtr> args, Frame* frame)
 	return f->Invoke(&args, frame);
 	}
 
+// The same, but raises an interpreter exception if the function does
+// not return a value.  Used for calls inside "when" conditions.  The
+// last argument is the address of the calling function; we just need
+// it to be distinct to the call, so we can associate a Trigger cache
+// with it.
+extern ValPtr when_invoke__CPP(Func* f, std::vector<ValPtr> args, Frame* frame, void* caller_addr);
+
+// Thrown when a call inside a "when" delays.
+class DelayedCallException : public InterpreterException
+	{
+public:
+	DelayedCallException() { }
```

Comment:
No need to do this. It'll get automatically generated. If you're set on it being there, change `{}` to `= default`.

Reply:
Got it, thx.  All of the above now addressed with the latest pushes (which also include unrelated NEWS items, for convenience).